templates/test/ci/cluster-template-prow-multi-tenancy.yaml

```diff
@@ -418,6 +418,18 @@ data:
                             type: string
                         type: object
                       type: array
+                    serviceLoadBalancerIPs:
```

Comment:
can we completely get rid of the multi-tenancy flavor now that all flavors are using multi-tenancy?

Reply:
I removed it from templates and removed the test, missed this one